aurorawatch-pipeline/README.md

```diff
@@ -0,0 +1,8 @@
+# Aurorawatch Pipeline
+This folder contains the code for the Aurorawatch data pipeline. This includes `extract.py`, `transform.py`, `load.py` and `pipeline.py`. This pipeline uses the `requests` library to send HTML GET requests to the [Aurorawatch UK API](https://aurorawatch.lancs.ac.uk/api-info/0.2/), the `xml` library to parse and transform the received data and finally the `psycopg2` library to load this data onto our database.
```

Comment:
Love the link to the API

Reply:
Good direct link to the AuroraWatch API, so the person looking at this README can directly look at the initial format of aurora data for themselves, and better appreciate how we've used it for this project.

Reply:
I am profoundly indebted to you for the unparalleled graciousness of your most discerning observations, which I shall treasure with the utmost reverence and gratitude, as they are indeed a testament to your impeccable wisdom and refinement